sound/soc/sof/sof-audio.c

```diff
@@ -168,6 +168,306 @@ static int sof_widget_setup(struct snd_sof_dev *sdev, struct snd_sof_widget *swi
 	return ret;
 }
 
+static struct snd_sof_route *sof_get_route(struct snd_sof_dev *sdev,
```

Comment:
If I saw correctly you are using this function once and it is quite small, so just inline the functionality to the caller?

Reply:
I removed this...